app/code/Magento/InventoryLowQuantityNotificationApi/Test/Api/GetSourceItemConfigurationTest.php

```diff
@@ -23,8 +23,8 @@ class GetSourceItemConfigurationTest extends WebapiAbstract
      */
     public function testGetSourceItemConfiguration()
     {
-        $sourceCode = 'eu-1';
-        $sku = 'SKU-1';
+        $sourceCode = 'eu-2';
+        $sku = 'SKU-3';
```

Comment:
what's the reason to change source code / sku in this case? 

as we have
```
    [
        SourceItemInterface::SOURCE_CODE => 'eu-1',
        SourceItemInterface::SKU => 'SKU-1',
        SourceItemInterface::QUANTITY => 5.5,
        SourceItemInterface::STATUS => SourceItemInterface::STATUS_IN_STOCK,
    ],
```
in our fixture data

Reply:
@maghamed, replaced. But with qty=5.6. As it now in fixture.